src/js/angular/core/directives/queryeditor/query-editor.directive.js

```diff
@@ -768,6 +768,15 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
                 var queryResultValue = queryResultElement.innerText.substring(1, queryResultElement.innerText.length - 1);
                 queryResultElement.innerHTML = "";
                 YASQE.runMode(queryResultValue, "sparql11", document.getElementById("highlighted_output"));
+
+                var explainPlanMetrics = document.getElementsByClassName("cm-comment");
```

Comment:
This will get all elements with class "cm-comment" the page. Better narrow the scope of the selector to yasr only, i.e something like $(".yasr .cm-comment") 
Also, better use jquery API instead of the raw Web API.